k8s/sonarqube/README.md

```diff
@@ -0,0 +1,439 @@
+  digest=$(docker pull ${!i} | sed -n -e 's/Digest: //p');
+  export $i="$repo@$digest";
+  env | grep $i;
+done
+```
+
+Create a certificate for PostgreSQL. If you already have a certificate that you
+want to use, copy your certificate and key pair in to the `server.crt` and
+`server.key` files.
+
+```shell
+# create a certificate for postgresql
+openssl req -x509 -nodes -days 365 -newkey rsa:2048 \
+    -keyout server.key \
+    -out server.crt \
+    -subj "/CN=postgresql/O=postgresql"
+
+kubectl --namespace $NAMESPACE create secret generic $APP_INSTANCE_NAME-tls \
+        --from-file=./server.crt --from-file=./server.key
+```
+
+Generate random password for PosgreSQL:
+
+```shell
+export POSTGRESQL_DB_PASSWORD=$(openssl rand 9 | openssl base64 -A | openssl base64)
+```
+
+Enable Stackdriver Metrics Exporter:
+
+> **NOTE:** Your GCP project should have Stackdriver enabled. For non-GCP clusters, export of metrics to Stackdriver is not supported yet.
+
+By default the integration is disabled. To enable, change the value to `true`.
+
+```shell
+export METRICS_EXPORTER_ENABLED=false
+```
+
+#### Expand the manifest template
+
+Use `helm template` to expand the template. We recommend that you save the
+expanded manifest file for future updates to the application.
+
+```shell
+helm template chart/sonarqube \
+  --name="$APP_INSTANCE_NAME" \
+  --namespace="$NAMESPACE" \
+  --set sonarqube.image=$IMAGE_SONARQUBE \
+  --set postgresql.image=$IMAGE_POSTGRESQL \
+  --set postgresql.exporter.image=$IMAGE_POSTGRESQL_EXPORTER \
+  --set postgresql.db.password=$DB_PASSWORD  \
+  --set metrics.image=$METRICS_EXPORTER_ENABLED > ${APP_INSTANCE_NAME}_manifest.yaml
+```
+#### Apply the manifest to your Kubernetes cluster
+
+Shell command for apply manifest
+```shell
+kubectl apply -f "${APP_INSTANCE_NAME}_manifest.yaml" --namespace "${NAMESPACE}"
+```
+
+#### View the app in the Google Cloud Console
+
+To get the Console URL for your app, run the following command:
+
+```shell
+echo "https://console.cloud.google.com/kubernetes/application/${ZONE}/${CLUSTER}/${NAMESPACE}/${APP_INSTANCE_NAME}"
+```
+
+To view the app, open the URL in your browser.
+
+# Using the SonarQube Community Edition
+
+#### View the application in the Google Cloud Console
+
+To get the Console URL for your application, run the following command:
+
+```shell
+echo "https://console.cloud.google.com/kubernetes/application/${ZONE}/${CLUSTER}/${NAMESPACE}/${APP_INSTANCE_NAME}"
+```
+Application is not exposed to external world. To get access to SonarQube run the following command:
+```bash
+    kubectl port-forward \
+      --namespace $NAMESPACE \
+      svc/$APP_INSTANCE_NAME-sonarqube-svc \
+      9000:9000
+```
+SonarQube will be available on `localhost:9000`. All interaction with application goes  through `9000` port. Cli also will be available.
+
+To get access web-page with default credentials:
+```bash
+http://localhost:9000
+Login: admin
+Password: admin
+```
+
+# Application metrics
```

Comment:
We already have polished text by our Tech Writer. Can you follow https://github.com/GoogleCloudPlatform/click-to-deploy/blob/f307c0ae16f8a6aa959d9c70f9a92c71b565553b/k8s/postgresql/README.md#application-metrics?